### PR TITLE
PRM - Fungii

### PR DESCRIPTION
**Pull request by pisskop**
Adds funguses

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
"Fungi"?

**Comment by pisskop** [comment]
Mooses.


Ill change the title.

File: data/mods/PKs_Rebalance/pk_critter_fungus.json [hunk 1]
```diff
@@ -0,0 +1,938 @@
...
+    "melee_cut":4,
+    "dodge":2,
+    "armor_bash":0,
+    "armor_cut":0,
+    "vision_day":50,
+    "vision_night":10,
+    "hp":50,
+    "death_function":["FUNGUS", "FUNGUS", "FUNGUS", "FUNGUS", "FUNGUS", "DISINTEGRATE", "SMOKEBURST"],
+    "special_attacks":[["LUNGE", 3], ["SUICIDE", 90], ["SHRIEK", 15], ["FUNGUS", 60]],
+    "description":"A deer brought into the fold of the Mcyus.  Suicidally fast.  So fast you hardly have time to recognize its affliction.",
```
Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_critter_fungus.json]: "Mcyus" - typo?

File: data/mods/PKs_Rebalance/pk_critter_fungus.json [hunk 2]
```diff
@@ -0,0 +1,938 @@
...
+    "death_function":["NORMAL", "FUNGUS"],
+    "special_attacks":[
+        ["FUNGUS", 180], ["SHRIEK", 120],
+        {
+            "type" : "bite",
+            "cooldown" : 20,
+            "min_mul" : 1.0
+        },
+        ["GRAB", 15]],
+    "description":"An apparent survivor of the apocolypse, this individual has visibly active fungal growth sprouting thoughout their body.  Whatever individuality they once held is completely subverted.",
```
Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_critter_fungus.json]: "apocolypse" - typo?

File: data/mods/PKs_Rebalance/pk_critter_fungus.json [hunk 3]
```diff
@@ -0,0 +1,938 @@
...
+            "move_cost" : 105,
+            "targeting_cost" : 50,
+            "range" : 13,
+            "description" : "The infected survivor fires wildly!",
+            "targeting_sound" : "incoherent gibberish!",
+            "targeting_volume" : 15,
+            "no_ammo_sound" : "a chk!"
+        }],
+    "starting_ammo" : {"9mm": 31},
+    "description":"A survivor of the apocolypse, this individual is succumbbing to the mycus, paranoia, and incoherency.  Unpredicable and terminally-ill, they will not respond to any attempts to communicate.  You can't help but notice they still wear their armor and holster, despite the fresh fungal budding that threatens to rip through it  . . .",
```
Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_critter_fungus.json]: "apocolypse", "succumbbing"

Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_critter_fungus.json]: Also, replacing of "  . . ." by a proper ellipsis symbol (…) could be probably better.

**Comment by pisskop** [comment]
Mycus is a nono word?

File: data/mods/PKs_Rebalance/pk_critter_fungus.json [hunk 4]
```diff
@@ -0,0 +1,938 @@
...
+            "targeting_cost" : 140,
+            "range" : 4,
+            "description" : "KABLAAM!!",
+            "targeting_sound" : "the barrel snapped into place!",
+            "targeting_volume" : 6,
+            "no_ammo_sound" : "the hammer slap foreward with a hollow rapping!"
+        }
+    ],
+    "starting_ammo" : {"shot_00": 10},
+    "description":"A survivor of the apocolypse, this individual is succumbbing to the mycus, paranoia, and incoherency.  Unpredicable and terminally-ill, they will not respond to any attempts to communicate.  You can't help but notice they still wear their armor and holster, despite the fresh fungal budding that threatens to rip through it  . . .",
```
Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_critter_fungus.json]: same typos as previously.

**Comment by Rivet-the-Zombie** [comment]
>Mycus is a nono word?

What do you mean?

File: data/mods/PKs_Rebalance/pk_critter_fungus.json (hunk 3 above)
Comment (Rivet-the-Zombie) [inline data/mods/PKs_Rebalance/pk_critter_fungus.json]: >Also, replacing of " . . ." by a proper ellipsis symbol (…) could be probably better.

Or just a period.

Also:
Unpredicable -> Unpredictable
terminally-ill -> terminally ill

**Comment by pisskop** [comment]
i meant he flagged it as wrong or bugged - is that from a desire to not use the term, or not use it casually?

ive removed references already

**Comment by Coolthulhu** [comment]
Pretty sure it was just about the typo - "Mcyus"

**Comment by Cyrano7** [comment]
@pisskop Does you text editor have a spell check option? I've notice that in the merged material for your mod some misspellings have slipped through even.

**Comment by illi-kun** [comment]
"Mcyus" is a nono word (as all other words with typos in their spelling), "Mycus" is just fine. All my comments are about typos, I didn't check anythink else in this PR. Sorry I was unclear.

File: data/mods/PKs_Rebalance/pk_critter_fungus.json [hunk 5]
```diff
@@ -0,0 +1,938 @@
...
+    "vision_night":5,
+    "hp":85,
+    "death_function":"NORMAL",
+    "special_attacks":[["FUNGUS", 60],
+        {
+            "type" : "bite",
+            "cooldown" : 3
+        }
+    ],
+    "description":"A large, astoundingly tough creature that used to be a dog or boar.  Its massive maw takes up almost half its body. Rows of teeth pushing older rows forward; creating a jutting smile only made more terrifying by the beady red eyes, socketed above behind the maw, looking down at its prey.",
```
Comment (illi-kun) [inline data/mods/PKs_Rebalance/pk_critter_fungus.json]: Please use two spaces after period here: ```its body. Rows```

**Comment by pisskop** [comment]
Its fine.  Im not always concious of my spelling choices.

**Comment by pisskop** [comment]
is this current correcred version acce

**Comment by pisskop** [comment]
I dingle-dongled with this rebase, but Im not even sure where I enter the rebase.

**Pull request closed without merging**